ingest/pipeline/definition.yml

```diff
@@ -74,3 +77,35 @@ apm_error_grouping_name:
       field: error.grouping_name
       copy_from: error.log.message
       if: ctx.error?.log?.message != null
+
+apm_opentelemetry_metrics:
+  description: Populate Elastic APM metric fields from well-known OpenTelemetry metric counterparts
+  processors:
+  - set:
+      # Copy `runtime.jvm.memory.area` (OpenTelemetry) to `jvm.memory.{area}.{type}` (Elastic APM).
+      field: jvm.memory.{{labels.area}}.{{labels.type}}
+      copy_from: runtime.jvm.memory.area
+      if: ctx.runtime?.jvm?.memory?.area != null && ctx.labels?.area != null && ctx.labels?.type != null
+  - set:
+      # Copy `runtime.jvm.gc.time` (OpenTelemetry) to `jvm.gc.time` (Elastic APM).
+      # Both are defined in milliseconds.
+      field: jvm.gc.time
+      copy_from: runtime.jvm.gc.time
+      if: ctx.runtime?.jvm?.gc?.time != null
+  - set:
+      # Copy `runtime.jvm.gc.count` (OpenTelemetry) to `jvm.gc.count` (Elastic APM).
+      field: jvm.gc.count
+      copy_from: runtime.jvm.gc.count
+      if: ctx.runtime?.jvm?.gc?.count != null
+  - set:
+      # Copy `runtime.jvm.gc.collection` (OpenTelemetry) to `jvm.gc.time` (Elastic APM).
+      # Both are defined in milliseconds. This is the old name for `runtime.jvm.gc.time`.
+      field: jvm.gc.time
+      copy_from: runtime.jvm.gc.collection
+      if: ctx.runtime?.jvm?.gc?.collection != null
```

Comment:
nit: It probably shouldn't happen, but could `runtime.jvm.gc.time` and `runtime.jvm.gc.collection` both be set? If so the value from the _old_ name would override the newer one. I'd probably flip it, or set `override: false` here.

Reply:
Fair, I'll move `runtime.jvm.gc.collection` above `runtime.jvm.gc.time`.